sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractRulesDefinition.java

```diff
@@ -179,5 +211,14 @@ private static class SecurityStandards {
 
     @SerializedName("OWASP")
     String[] owasp2017 = {};
+
+    @SerializedName("PCI DSS 3.2")
+    String[] pciDss3_2 = {};
+
+    @SerializedName("PCI DSS 4.0")
+    String[] pciDss4_0 = {};
+
+    @SerializedName("ASVS 4.0")
```

Comment:
I haven't found a single `ASVS 4.0` in the RSPEC. The closest thing I see is `MASVS`. Is this surely the correct value to use?

Reply:
Yes, you can check for example `S2068`:
https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/rspec/cs/S2068_c%23.json#L35-L39

Reply:
Ok, I was on a wrong branch in my local rspec